### DISCUSSION TITLE
Tick blocking with Biomes O' Plenty

### DISCUSSION THREAD

**Discussion by XvaleX10** [ACGaming/UniversalTweaks#350] [Bugfix Suggestions]
[crash-2023-08-13_15.43.52-server.txt](https://github.com/ACGaming/UniversalTweaks/files/12329663/crash-2023-08-13_15.43.52-server.txt)
Could you fix it?

**Comment by ACGaming** [comment]
Need more information on this, is this an issue introduced by Universal Tweaks? If so, please try to pinpoint the tweak causing it.

**Comment by XvaleX10** [comment]
> Need more information on this, is this an issue introduced by Universal Tweaks? If so, please try to pinpoint the tweak causing it.

Universal tweak is not the cause, I think it's a bug of the mod itself never fixed, because it happened to me many other times in different modpacks, I was wondering if it could be fixed by your mod since it already makes some changes to Biomes O' Plenty.

**Comment by ACGaming** [comment]
Can you determine when this occurs or is it seemingly random?

**Comment by XvaleX10** [comment]
It usually happens when I try to generate new chunks, suddenly the ticks freeze and I can only move in those already loaded chunks

**Comment by XvaleX10** [comment]
> [crash-2023-08-13_15.43.52-server.txt](https://github.com/ACGaming/UniversalTweaks/files/12329663/crash-2023-08-13_15.43.52-server.txt) Could you fix it?

Sometimes after a few minutes it starts working again, while other times like this the whole game crashes, in the task manager it says not responding

**Comment by ACGaming** [comment]
Have a feeling this could be modpack related or do you have a fitting BoP GitHub issue for me?

**Comment by jchung01** [comment]
Unlikely to be a BoP issue according to Glitchfiend/BiomesOPlenty#1286, only shows BoP in the report as the world type and thus chunk generator used is BoP. latest.log and/or debug.log may be more helpful. Other than that, it would have to be reproduceable in a more isolated environment to know what the issue is. 

I can see that you have RLMixins installed, which may or may not have issues with MixinBooter 8, so another suggestion is downgrading to MixinBooter 7 and using the MB7 version of UT to see if the issue persists.

**Comment by XvaleX10** [comment]
[crash-2024-02-03_11.56.27-server.txt](https://github.com/ACGaming/UniversalTweaks/files/14151154/crash-2024-02-03_11.56.27-server.txt)
